pkg/client/channel/api.go

```diff
@@ -50,6 +50,7 @@ type Request struct {
 	// The invoked chaincode (specified by ChaincodeID) may optionally be added to the invocation
 	// chain along with any collections, otherwise it may be omitted.
 	InvocationChain []*fab.ChaincodeCall
+	IsInit          bool
```

Comment:
@bstasyszyn what do you think of IsInit added to several non-test structures?

Reply:
Looks like this is a new field for 2.0: https://github.com/hyperledger/fabric/blob/527fbdc1328d181dc651eaedbe91d51d3487c82b/core/chaincode/chaincode_support.go#L228

… something I missed. Now you need to explicitly call Init on your chancode.